src/main/rx/msp.c

```diff
@@ -71,6 +73,16 @@ static uint8_t rxMspFrameStatus(rxRuntimeState_t *rxRuntimeState)
     return RX_FRAME_COMPLETE;
 }
 
+uint8_t rxMspOverrideFrameStatus(void)
+{
+    if (!rxMspOverrideFrameDone) {
+        return RX_FRAME_PENDING;
+    }
+
+    rxMspOverrideFrameDone = false;
+    return RX_FRAME_COMPLETE;
+}
```

Comment:
```suggestion
#if defined(USE_RX_MSP_OVERRIDE)
uint8_t rxMspOverrideFrameStatus(void)
{
    if (!rxMspOverrideFrameDone) {
        return RX_FRAME_PENDING;
    }

    rxMspOverrideFrameDone = false;
    return RX_FRAME_COMPLETE;
}
#endif
```

Reply:
Right, It's a very good point that it's already gated, because MSP override mode needs to be enabled in the console. 

Regard the suggested change, this condition is already checked on this line https://github.com/betaflight/betaflight/blob/99ba203646ebc6881785033da42348dd19470521/src/main/rx/rx.c#L569

This function would not be executed when MSP_OVERRIDE is not enabled in the console. But it would not hurt to have the extra condition here as well. So I have applied the suggested change.

The only thing I'm worried a bit is the unused variable `rxMspOverrideFrameDone ` in case the condition is not true. I hope it would not complain about it on the build stage (or that it's checked in the CI)

Reply:
Have tested the build without `MSP_RX_OVERRIDE_MSP` and it builds without issues.

Reply:
> Have tested the build without `MSP_RX_OVERRIDE_MSP` and it builds without issues.

https://github.com/betaflight/betaflight/pull/13380/files#diff-bf96d9585f75b1424f3fa4d446b03998efe43463c56458d5391e53397dc2d64eR61 is preventing unused warning